diff --git a/node_modules/react-native/Libraries/NativeComponent/ViewConfigIgnore.js b/node_modules/react-native/Libraries/NativeComponent/ViewConfigIgnore.js
index 0000000..1111111 100644
--- a/node_modules/react-native/Libraries/NativeComponent/ViewConfigIgnore.js
+++ b/node_modules/react-native/Libraries/NativeComponent/ViewConfigIgnore.js
@@ -35,7 +35,7 @@
  * @flow strict
  */
 export function ConditionallyIgnoredEventHandlers<
-  const T: {+[name: string]: true},
+  T: {+[name: string]: true},
 >(value: T): T | void {
   if (Platform.OS === 'ios') {
     return value;
